src/windows/GeolocationProxy.js

```diff
@@ -14,47 +14,43 @@
  * limitations under the License.
  */
 
-var PositionError = require('./PositionError'),
-    ids = {},
-    loc;
+var PositionError   = require('./PositionError');
+var ids             = {};
```

Comment:
Nitpick: `ids` is not a very descriptive name. This object holds on to subscribed callbacks, so maybe rename it to something related to that?